applications/tari_walletd/web_ui/src/routes/Transactions/TransactionDetails.tsx

```
@@ -33,18 +33,16 @@ import Events from "./Events";
 import Logs from "./Logs";
 import Instructions from "./Instructions";
 import Substates from "./Substates";
+import Inputs from "./Inputs";
+import Signers from "./Signers";
+import ExecutionResults from "./ExecutionResults";
+import FeeReceipt from "./FeeReceipt";
 import StatusChip from "../../Components/StatusChip";
 import KeyboardArrowDownIcon from "@mui/icons-material/KeyboardArrowDown";
 import KeyboardArrowUpIcon from "@mui/icons-material/KeyboardArrowUp";
 import Loading from "../../Components/Loading";
 import Error from "../../Components/Error";
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)